llvm/test/CodeGen/PowerPC/aix-tls-xcoff-reloc-large.ll

```diff
@@ -32,7 +33,7 @@ entry:
 ; RELOC-NEXT:   Section (index: 1) .text {
 ; RELOC-NEXT:   Relocation {
 ; RELOC-NEXT:     Virtual Address: 0x16
-; RELOC-NEXT:     Symbol: .TIInit (17)
+; RELOC-NEXT:     Symbol: TIInit (19)
```

Comment:
I know this is not this patch's scope. But it may make more sense to use a RE to handle the symbol index. We were trying to fix this, but I guess we didn't change them all.